syntax = "proto3";

service Data {
    rpc SendWebhookData(DataRequest) returns (DataResponse);
}


message DataRequest {
    string model_name = 1;
    string webhook_url = 2;
}


message DataResponse {
    string json_data = 1;
}